app/views/admin/environments/index.html.erb

```diff
@@ -21,7 +21,7 @@
             <td>
               <%= link_to "Edit", edit_admin_environment_path(env) %>
               |
-              <%= link_to "Delete", admin_environment_path(env), method: :delete, data: { confirm: "Are you sure?" } %>
+              <%= link_to_delete(admin_environment_path(env)) %>
```

Comment:
Couldn't find any docs so had to check the source, this magic comes from `url_for` which is also used in redirects. I'll go through and make these as consistent as I can.